sdk/storage/storage-file/samples/typescript/advanced.ts

```diff
@@ -14,7 +14,7 @@ import {
   ShareClient,
   FileServiceClient,
   StorageClient
-} from "../.."; // Change to "@azure/storage-file" in your package
+} from "../../src"; // Change to "@azure/storage-file" in your package
```

Comment:
is this change needed?

Reply:
Previously, it used to throw linter errors without this.
I think we should standardize on what extensions to use.

Reply:
we have format and linting npm scripts. I'd suggest be consistent with configuration used by them. We can discuss whether or not a rule is good.